procrustes/test/test_psdp.py

```diff
@@ -66,17 +70,26 @@ def test_psdp_generic_square():
         ]
     )
     assert_almost_equal(s, actual_result)
-    assert_almost_equal(error, 5.600999068630569, decimal=3)
+    assert_almost_equal(error, 31.371190566800497, decimal=3)
+    res = psdp_peng(a=a, b=b)
+    s, error = res["s"], res["error"]
+    actual_result = np.array(
+        [
+            [0.1440107, -0.05853613, 0.00939016],
+            [-0.05853613, 0.02379322, -0.00381683],
+            [0.00939016, -0.00381683, 0.00061228],
+        ]
+    )
+    assert_almost_equal(s, actual_result)
+    assert_almost_equal(error, 33.43617791613022, decimal=3)
```

Comment:
The same as above.

Reply:
fixed in [8c7015b](https://github.com/theochem/procrustes/pull/187/commits/8c7015b5413c07718ecf7cd817f58568144fa3ae)